### PR TITLE
Edit superset create-admin and initialization commands

### LINKED ISSUE #43: Edit role to work with superset 0.34.0
The current role has commands that have been deprecated for Superset 0.34.0 release.

- The app name is now added as a variable by exporting it to `FLASK_APP`
- `fabmanager` has been replaced with `flask fab`
- The runserver command has been replaced with `run`

All these changes are Apache superset docs https://superset.incubator.apache.org/installation.html#superset-installation-and-initialization.

We could look into the option of running the server using `gunicorn` that works for all superset versions https://superset.incubator.apache.org/installation.html#superset-installation-and-initialization

### PR DESCRIPTION
**Pull request by nateMJ**
Fixes #43 for superset version 0.34.0

### CODE REVIEW

File: tasks/install.yml [hunk 1]
```diff
@@ -103,7 +103,7 @@
   become: true
   become_user: "{{ superset_user }}"
   pip:
-    name: superset
+    name: apache-superset
```
Comment (jasonrogena) [inline tasks/install.yml]: @nateMJ could we convert this into a variable with the default value being "apache-superset". I think we might also want to update the default superset version to `0.34.0`.

File: templates/superset.service.j2 [hunk 2]
```diff
@@ -12,7 +12,7 @@ Environment="PATH={{ superset_virtualenv_path }}/bin:$PATH"
 Environment="PYTHONPATH={{ superset_python_path }}"
 User={{ superset_user }}
 Type=simple
-ExecStart={{ superset_virtualenv_path }}/bin/superset runserver -p {{ superset_port }} -w {{ superset_workers }}
+ExecStart={{ superset_virtualenv_path }}/bin/superset run -p {{ superset_port }} --with-threads
```
Comment (jasonrogena) [inline templates/superset.service.j2]: @nateMJ with the new command, are we able to specify the number of threads/workers?

File: tasks/config.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@
 - name: Create an admin user.
   become: true
   become_user: "{{ superset_user }}"
-  shell: "export PYTHONPATH='{{ superset_python_path }}' && {{ superset_virtualenv_path }}/bin/fabmanager create-admin --app {{ superset_app }} --username {{ superset_username }} --firstname {{ superset_firstname }}  --lastname {{ superset_lastname }}  --email {{ superset_email }} --password {{ superset_pass }}"
+  shell: "export PYTHONPATH='{{ superset_python_path }}' && export FLASK_APP='{{ superset_app }}' && {{ superset_virtualenv_path }}/bin/flask fab create-admin --username {{ superset_username }} --firstname {{ superset_firstname }}  --lastname {{ superset_lastname }}  --email {{ superset_email }} --password {{ superset_pass }}"
```
Comment (jasonrogena) [inline tasks/config.yml]: @nateMJ assuming this command is not available in older versions of superset, let's say 0.29.0; do you think 1. we should add backwards compatibility for installing these versions and 2. that a good way of allowing backwards compatibility would be to put the commands that have changed into variables, and setting the default values to those supported in 0.34.0?

File: templates/superset.service.j2 (hunk 2 above)
Comment (nateMJ) [reply templates/superset.service.j2]: Unfortunately not although it's possible using gunicorn. I'm not sure if we want to use that over flask commands

File: tasks/config.yml (hunk 3 above)
Comment (nateMJ) [reply tasks/config.yml]: Yes, they are not compatible with the older superset versions. I agree with approach 2. I will change them to variables

File: templates/superset.service.j2 (hunk 2 above)
Comment (jasonrogena) [reply templates/superset.service.j2]: Makes sense. Just checked and the superset_workers variable is used in the systemd unit file for the service.

**Pull request merged**